rholang/src/main/scala/coop/rchain/rholang/interpreter/error/package.scala

```diff
@@ -0,0 +1,27 @@
+package coop.rchain.rholang.interpreter
+
+import cats.implicits._
+import cats.{Applicative, Functor, Monad}
+import cats.mtl.{ApplicativeAsk, FunctorTell, MonadState}
+
+package object error {
+
+  type _error[F[_]] = ApplicativeAsk[F, Option[Throwable]] with FunctorTell[F, Throwable]
+
+  def _error[F[_], E](implicit ev: _error[F]): _error[F] = ev
+
+  def errorHandler[F[_]: Monad, E](
```

Comment:
where is this used? 

We are adding RWLock and justify it by this function but we don't have a usage of this function. The bottom-up approach can lead to unnecessary constructions. I would like to see top-down how this is used.